src/Neo/SmartContract/Native/NeoToken.cs

```diff
@@ -572,19 +576,23 @@ public class NeoAccountState : AccountState
             /// </summary>
             public ECPoint VoteTo;
 
+            public BigInteger LastGasPerVote;
+
             public override void FromStackItem(StackItem stackItem)
             {
                 base.FromStackItem(stackItem);
                 Struct @struct = (Struct)stackItem;
                 BalanceHeight = (uint)@struct[1].GetInteger();
                 VoteTo = @struct[2].IsNull ? null : ECPoint.DecodePoint(@struct[2].GetSpan(), ECCurve.Secp256r1);
+                LastGasPerVote = @struct[3].GetInteger();
```

Comment:
This will fork the chain

Reply:
Yes and no. We'll get a different NEO contract state, true. We may have some problem with `getAccountState` method, but that's easy to solve (strip the third struct item, return only the old two ones to stay compatible, we can add `getAccountStateV2` that will return everything, but we might as well not do that, I don't see a lot of uses for this data). But anything else will just work: balances, transfer data, GAS calculations, etc. So it looks more like a regular DB resync to me.